manager/config/crd/bases/app.m4d.ibm.com_blueprints.yaml

```diff
@@ -68,10 +68,32 @@ spec:
                                   format:
                                     description: Format represents data format (e.g. parquet) as received from catalog connectors
                                     type: string
+                                  vault:
```

Comment:
Should Vault or all the fields should repeat for each action in the blueprint? could some part be factored out?

Reply:
The `authPath` should be different for every cluster. The `address` will probably be the same for each cluster. I'm not sure how we will configure `role`. But the `role` and `secretPath` might be the changing factor per blueprint.

Reply:
I can see a far future where the address is different (e.g., when using on-prem vaults). Anyway, for what we have defined we can move address, authPath, and role one level up. Not sure if that's that important though. It should be decided by what sounds more convenient for module developers @cdoron

Reply:
@roee88 : I think we could keep the current configuration as is, especially if this may also be the way to go for the "far future". We could always re-examine this approach if needed.